release-notes-v1.0.1.md

```diff
@@ -0,0 +1,28 @@
+v1.0.1
+
+### Fixed Issues
+
+**pri-fidoiot**, **test-fidoiot**: The version of third-party dependencies have been updated to resolve [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) vulnerability.   
+**client-sdk-fidoiot**: There are no changes from previous release.  
+
+
+### SHA256 checksum for release binaries
+
+*Following SHA256 checksum is calculated using sha256sum tool*
+```
+
+```
+
+### SHA256 checksum for release binaries
```

Comment:
Please remove the duplicate section.
Checksum for NOTICES:
01cedccc4f5040cbf5f5d1154366fbc9d2e5af5a58ac3c4ef5cd4d587e6f30a8  NOTICES-v1.0.1.tar.gz